### PR TITLE
docker/deploy

### PR DESCRIPTION
**Pull request by roadscape**

### CODE REVIEW

File: hive/indexer/cli.py [hunk 1]
```diff
@@ -26,7 +27,8 @@ def indexer():
 def index_from_steemd():
     """import blocks from .json.lst file"""
     echo('Loading blocks from steemd...')
-    sync_from_steemd()
+    setup()
+    sync_from_steemd(True)
```
Comment (roadscape) [inline hive/indexer/cli.py]: `index_from_steemd` is outdated, if it's meant to be the main "launch daemon" command, let's rename it and just have it call `run()` method in hive.indexer.core, runs `setup()` only if needed, handles multiple sync strategies, and then streams blocks forever.

File: hive/db/schema.py [hunk 2]
```diff
@@ -230,19 +231,26 @@ def setup(connection_url=_url):
...
 
-    # Insert hive_accounts data
-    insert = hive_accounts.insert()
-    conn.execute(insert, [
-        {'name': 'miners', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'null', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'temp', 'created_at': '1970-01-01T00:00:00'},
-        {'name': 'initminer', 'created_at': '1970-01-01T00:00:00'}
-    ])
+    except IntegrityError:
```
Comment (roadscape) [inline hive/db/schema.py]: Why would we get an IntegrityError here?

Comment (john-g-g) [inline hive/db/schema.py]: I think it happens if the db already exists? It was always block 000000000000 that was giving the error

Comment (roadscape) [inline hive/db/schema.py]: Ok, should be safe to remove this since `setup()` is only called once in `run()` (it's skipped if any tables exist).

**Pull request merged**